style.css

```diff
@@ -490,3 +508,434 @@ input[type="submit"]:disabled {
+    background-repeat: no-repeat;
+    color: var(--txtcolor);
+  }
+
+  #work > .cartt2 > .cart > h2 {
+    padding: 2.3rem 1rem;
+    font-weight: 500;
+    font-family: 'Crete Round', serif;
+    font-size: 32px;
+    line-height: 40px;
+  }
+
+  #work > .cartt2 > #c6 {
+    display: flexbox;
+    flex-direction: column;
+    margin-top: 2rem;
+    margin-bottom: 1rem;
+    padding-top: 2.5rem;
+    height: 20rem;
+    width: 19.5rem;
+    left: 0;
+    top: 0;
+    border: none;
+    border-radius: 0;
+    background-image: url(Resources/Images/Mask-Group2.png);
+    background-position: center center;
+    background-repeat: no-repeat;
+    padding-bottom: 0%;
+    color: var(--txtcolor);
+  }
+
+  #about > button {
+    margin-left: 4.5rem;
+  }
+
+  #work .cartt2 > .cart > button {
+    height: 3rem;
+    margin-top: 1rem;
+    margin-bottom: 0;
+    margin-left: 0;
+    margin-right: 0;
+    width: 19.5rem;
+    border-radius: 0%;
+    border: none;
+    font-family: 'Inter', sans-serif;
+    font-size: 17px;
+    font-weight: 700;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    text-align: center;
+    color: var(--txtcolor);
+    background-color: var(--maincolor);
+  }
+
+  #work > .cartt2 > #c1 > button {
+    margin-top: 17rem;
+  }
+
+  #work > .cartt2 > #c3 > h2 {
+    padding: 3.5rem 1rem;
+  }
+
+  #work > .cartt2 > #c6 > h2 {
+    padding: 3.5rem 1rem;
+  }
+
+  #work > .cartt2 > .cart > h1 > span {
+    display: block;
+  }
+
+  #about > p {
+    width: 30rem;
+    margin-left: 4.5rem;
+  }
+
+  #work > .cartt2 > .cart > p {
+    font-family: 'Inter', sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    padding: 0 1rem;
+    letter-spacing: 0;
+    text-align: left;
+  }
+
+  #about > .listing > ul {
+    display: block;
+    flex-direction: column;
+    margin: 5rem 10rem;
+    font-size: larger;
+  }
+
+  #work > .cartt2 > .cart > ul {
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    padding: 0 1rem;
+  }
+
+  #work > .cartt2 > .cart > ul > li {
+    display: flex;
+    font-family: 'Inter', sans-serif;
+    justify-content: center;
+    align-items: center;
+    width: auto;
+    height: 2rem;
+    padding: 0.5rem 1rem;
+    margin: 0.5rem 0.5rem 0.5rem 0;
+    background-color: var(--colorb);
+  }
+
+  .cartt2 {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: auto;
+    column-gap: 2.5rem;
+    row-gap: 0.5rem;
+    padding: 0%;
+    margin-left: -6%;
+  }
+
+  #work {
+    width: auto;
+    place-items: center;
+  }
+
+  #about {
+    background-image: url(Resources/Images/illustration-aboutme.png);
+    background-repeat: no-repeat;
+    background-size: 30rem;
+    background-position: 20em right;
+    background-position-y: 14ch;
+  }
+
+  #about > .line {
+    background: var(--backc);
+    border: none;
+    height: 2px;
+    width: 73.2rem;
+    margin: 31rem 1rem 5rem 4.5rem;
```

Comment:
- [ ] As suggested by the previous reviewer. I kindly suggest you reduce the margin on the left of your `div` line to so so that the listings will vertically fit into the line as shown in the design.

Here's how to:

https://user-images.githubusercontent.com/52934128/168448593-5610fda1-c383-44e3-8448-f9ab3e1db221.mp4